lib/src/components/modal/Modal.js

```diff
@@ -4,14 +4,13 @@ import { ModalFooter } from './ModalFooter';
 import { ModalHeader } from './ModalHeader';
 import { ModalBody } from './ModalBody';
 import { ModalFooterConfirm } from './ModalFooterConfirm';
-import { ModalHeaderTabs } from './ModalHeaderTabs';
 
 function Modal({ children, show, onHide, srContentLabel, size, keyboard }) {
   return (
     <ReactModal
       className={`Modal ${size}`}
       isOpen={show}
-      closeTimeoutMS={3000}
+      closeTimeoutMS={300}
```

Comment:
The `closeTimeOutMS` value is to allow the animation to finish for the closing modal. I had set it too high which was interfering with the dom for a few seconds after the modal closed.